packages/taquito-local-forging/src/protocols.ts

```diff
@@ -10,6 +10,7 @@ export enum ProtocolsHash {
   PtHangz2 = 'PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx',
   Psithaca2 = 'Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A',
   PtJakart2 = 'PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY',
+  PtKathman = 'PtKathmanYPEsBau83zRwXK19HAQRzZSj6pCXnvtyLZFjRxdQxt',
```

Comment:
It should be `PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg`.

Reply:
fixed